src/test/operators/materialize_test.cpp

```diff
@@ -0,0 +1,43 @@
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "../base_test.hpp"
+#include "gtest/gtest.h"
+
+#include "operators/materialize.hpp"
+#include "operators/table_wrapper.hpp"
+#include "storage/chunk_encoder.hpp"
+#include "storage/table.hpp"
+
+namespace opossum {
+
+class OperatorsMaterializeTest : public BaseTest {};
```

Comment:
You're not testing Materialization of ReferenceTables at all here, imo. TableWrapper will not create a ReferenceTable.